dreammall-earth/dreammall.earth
# refactor(other): DRY kubernetes chart

Motivation
----------
I found `envFrom` configmaps a lot easier to maintain. So this PR will DRY up our helmfile and deployment configurations.

How to test
-----------
1. Checkout this branch
2. `git reset origin/master --hard` temporarily
3. `cd infrastructure/helmfile`
4. `helmfile apply`
5. `git reset @{u} --hard`
6. `helmfile diff` (to check the differences)
7. `helmfile apply`
8. Your deployment should behave as expected
